src/accumulation/point_cloud_accumulator.py

```diff
@@ -20,10 +20,13 @@ def __init__(self,
         self.__dataset = dataset
 
     def merge(self,
+              scene_id: str,
```

Comment:
same should apply for gedi accumulator then, I could do it?

Reply:
We don't have `gedi accumulator`, right? We have only `gedi accumulation strategy`. Strategies don't work with frames they work with point clouds: a strategy merges merges `old` point cloud (i.e. accumulated) with a point cloud from the current frame. What do you think?

Reply:
For Gedi, we need to know which object pointcloud to register to, if we register them with chain logic, similar to geometric accumulation, we run the risk of accumulating registration error as well. Therefore, current registration workflow registers all the instance objects to a chosen "first".  To be able to differentiate between point clouds we need some info like frame_id

Reply:
I believe strategies should not make a distinction between frames. If you need to start from different frame or use a different frame we can modify `a constructor` + `logic` of the `point_cloud_accumulator` to change the behaviour as we want. We will still have "chain" merge logic even if we are going to merge the clouds across different frames, right?
I am not sure I get what you want to achieve. Can you, please, explain it to me again and we can try to work from there?

Reply:
explained offline. ideally we need the info of the "first" instance appearence

Reply:
agreed to pass the initial cloud as a parameter of `point cloud strategy`